### PR TITLE
Recipe / Add Native Ads Express

### PR DESCRIPTION
**Pull request by lgvalle**
### Scope of the PR

Adds a recipe on how to create and integrate the newish Native Ads Express.

### Reasons to exist

* Having a single guide to walk you through the process of creating and Advert and integrating it is more useful than checking separate documentation for each task. 
* Google documentation is not very clear about advert sizes. At first sight it looks like the sizes you define when creating an Ad is the size you need to use later when integrating it. 
This cookbook tries to clarify that.

### CODE REVIEW

**Comment by lgvalle** [comment]
You can better preview this doc here: https://github.com/novoda/android-demos/blob/963b36fff1344c09700db9e303724735c563be2c/Cookbook/add_native_express_adverts.md

File: Cookbook/add_native_express_adverts.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+The process is the same as loading a regular ad:
+
+1. In the activity, initialise MobileAds:
+
+```
+MobileAds.initialize(getApplicationContext(), getString(R.string.ad_app_id));
+```
+
+2. Load the view and build a request:
```
Comment (tasomaniac) [inline Cookbook/add_native_express_adverts.md]: This still looks like `1.` in the preview

image

File: Cookbook/add_native_express_adverts.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+
+![screen shot 2017-01-05 at 15 31 37](images/native_ads_express_5.png)
+
+
+## 3. Loading  a native express add:
+
+The process is the same as loading a regular ad:
+
+1. In the activity, initialise MobileAds:
```
Comment (tasomaniac) [inline Cookbook/add_native_express_adverts.md]: Put `MobileAds` into backticks maybe?

File: Cookbook/add_native_express_adverts.md (hunk 1 above)
Comment (lgvalle) [inline Cookbook/add_native_express_adverts.md]: Fixed here 74dcf57

File: Cookbook/add_native_express_adverts.md (hunk 2 above)
Comment (lgvalle) [inline Cookbook/add_native_express_adverts.md]: Fixed here Add backticks to code

**Comment by lgvalle** [comment]
Updated here 3da83b8 @tobiasheine

**Comment by lgvalle** [comment]
You are right @mr-archano . I added a "Reasons to exist" block into the description. We can use the same approach in upcoming recipes. Is good to write down the reasons behind writing it

**Pull request merged**